### ISSUE TITLE
Getting 500 error on /p/<box-name>/last

### ISSUE THREAD

**Issue by Bbbrinks** [phstc/putsbox#4]
I get 500 errors on some of the boxes i'm trying to use, for example:
http://preview.putsbox.com/p/mp-cp-account-user/last

I have no other real information about this, i just send an email and try to look it up using the link above and it gives me 500 error.

**Comment by phstc** [comment]
Hey @sterfpaul 

Thanks for reporting it 🍻 

I can't debug it right now, but I'm adding the log output, so we can investigate it even after the bucket expires 😄 

```
Started GET "/p/mp-cp-account-user/last"
Processing by EmailsController#show as HTML
  Parameters: {"subdomain"=>"preview", "token"=>"mp-cp-account-user", "id"=>"last"}
undefined method `token' for nil:NilClass
  Rendered inline template (0.4ms)
Completed 500 Internal Server Error in 10ms

RuntimeError (A template needs to have a virtual path in order to be refreshed):
  app/controllers/emails_controller.rb:18:in `block (2 levels) in show'
  app/controllers/emails_controller.rb:17:in `show'
```

**Comment by phstc** [comment]
Fixed all together with #5. The requested format was not available, not it shows a proper message when that happens.
